apps/examples/curl_test/curl_main.c

```diff
@@ -0,0 +1,242 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <string.h>
+#include <curl/curl.h>
+
+
+#define CURL_TASH_PRI      100
+#define CURL_TASH_STAKSIZE 10240
+#define CURL_SCHED_POLICY  SCHED_RR
+
+
+struct input_arg {
+	int  argc;
+	char **argv;
+};
+
+/****************************************************************************
+ * curl_main
+ ****************************************************************************/
+static void showusage(void)
+{
+	printf("\ncurl test Usage\n");
+	printf("curl [command] <parameters>\n");
```

Comment:
In linux, [ ] means optional.
Could you update this linux likely to understand easily?
```
@ubuntu:~$ cp --help
Usage: cp [OPTION]... [-T] SOURCE DEST
  or:  cp [OPTION]... SOURCE... DIRECTORY
  or:  cp [OPTION]... -t DIRECTORY SOURCE...
Copy SOURCE to DEST, or multiple SOURCE(s) to DIRECTORY.
```